src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/SimpleRecordsFetcherFactory.java

```diff
@@ -44,7 +45,7 @@ public GetRecordsCache createRecordsFetcher(GetRecordsRetrievalStrategy getRecor
                             .setDaemon(true)
                             .setNameFormat("prefetch-cache-" + shardId + "-%04d")
                             .build()),
-                    idleMillisBetweenCalls);
+                    idleMillisBetweenCalls, metricsFactory, operation);
```

Comment:
The operation doesn't need to be passed from the shardConsumer. It can be hardcoded out here.